src/bennu/executables/bennu-simulink-provider/main.cpp

```diff
@@ -348,6 +352,8 @@ int main(int argc, char** argv)
     sEndpoint.str = vm["server-endpoint"].as<std::string>();
     pEndpoint.str = vm["publish-endpoint"].as<std::string>();
     BennuSimulinkProvider bsp(sEndpoint, pEndpoint, debug);
```

Comment:
bsp is being initialized twice, remove the old initialization without publishRate